### PR TITLE
docs: update PR template [#000000000]

### PR DESCRIPTION
**Pull request by Kryvchun**
### For developers

Simplify PR template.

## Asana

No task.

Thank you!

### CODE REVIEW

**Review by mrksbnc — APPROVED** [review]
LGTM! 🚀

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -36,9 +30,15 @@ Feel free to delete this section.
...
-- [ ] Rebase on top of `main` branch before review and before merging.
+- [ ] Make the branch up to date with `main`.
 - [ ] Ensure that CI/CD passed.
 - [ ] Review the changes by yourself.
 - [ ] **Remove [this section](#checklist-before-review)**.
 
+## Checklist before merging
+
+- [ ] Make the branch up to date with `main`.
+- [ ] Rebase after merging #XX on the latest main: `git rebase --onto main "PREVIOUS_BRANCH_NAME"`
```
Comment (ghost) [inline .github/PULL_REQUEST_TEMPLATE.md:41]: `git rebase`? We are feeling brave… :grinning:

**Review by ghost — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Kryvchun) [reply .github/PULL_REQUEST_TEMPLATE.md:41]: It is in case of nested branching

**Review by MaksimYworldr — APPROVED** [review]
LGTM

**Review by sm-worldr — APPROVED** [review]

**Pull request merged**